consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/validation/RoundChangePayloadValidatorTest.java

```diff
@@ -139,21 +139,9 @@ public void roundChangeWithZeroTargetRoundFails() {
     assertThat(messageValidator.validate(signedPayload)).isFalse();
   }
 
-  @Test
-  public void roundChangeWithNegativeTargetRoundFails() {
```

Comment:
Why did you get rid of this test? Seems like a valid case to test

Reply:
Because you can't encode negative numbers into a scalar integer - so this literally failed when attempting the setup of the test.